scripts/install_environment.sh

```diff
@@ -160,6 +160,22 @@ function config_chronyd() {
   sudo firewall-cmd --zone=libvirt --add-port=123/udp
 }
 
+function config_nginx() {
+  echo "Config nginx"
+
+  cat <<EOF | sudo podman build --tag load_balancer:latest -
+FROM quay.io/centos/centos:8.3.2011
+RUN dnf install -y nginx
+RUN sed -i -e '/^http {/,\$d'  /etc/nginx/nginx.conf
+RUN sed -i -e '\$a stream {\ninclude /etc/nginx/stream.d/*.conf;\n}' -e '/^stream {/,\$d' /etc/nginx/nginx.conf
+CMD ["bash", "-c", "while /bin/true ; do (ps -ef | grep -v grep | grep -q nginx && nginx -s reload) || nginx ; sleep 60 ; done"]
+EOF
+  sudo podman rm -f load_balancer || /bin/true
+  sudo mkdir -p $HOME/.test-infra/etc/nginx/stream.d
```

Comment:
why $HOME/.test-infra? why not inside test-infra? and git ignore it?

Reply:
Seemed OK to me.  Since it is a machine wide configuration, it should be stored on machine level